Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:xenial
+MAINTAINER Jason Walker <jason.walker@wustl.edu>
+
+LABEL \
+    description="Image for using fgbio"
+
+RUN apt-get update -y && apt-get install -y \
+    ant \
+    apt-utils \
+    build-essential \
+    bzip2 \
+    default-jdk \
+    default-jre \
+    gcc-multilib \
+    git \
+    libncurses5-dev \
+    libnss-sss \
+    nodejs \
+    python-dev \
+    python-pip \
```

Comment:
Are `nodejs` and  `python` needed for this tool?  (There may be a few other packages in this list that this question applies to :smile:)